controllers/storagecluster/quickstart_constants.go

```diff
@@ -9,7 +9,7 @@ metadata:
   name: "getting-started-ocs"
 spec:
   version: 4.7
-  displayName: "Getting started with Openshift Container Storage"
+  displayName: "Getting started with OpenShift® Container Storage"
```

Comment:
We just use `OpenShift Container Storage` everywhere. Why is this different?

Reply:
Fixed.

Reply:
@umangachapagain probably we should keep the (R) for consistency across the quickstarts. The install tour has it in its displayName as UXD wanted it there (for reasons unknown to me) . Either we change it here or we'll have to remove it in console-operator